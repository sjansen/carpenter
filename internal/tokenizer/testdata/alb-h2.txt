h2 2018-07-02T22:23:00.186641Z app/my-loadbalancer/50dc6c495c0c9188 10.0.1.252:48160 10.0.0.66:9000 0.001 0.002 0.003 200 200 5 257 "GET https://10.0.2.105:773/ HTTP/2.0" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36  (KHTML, like Gecko) Chrome/51.0.2704.103 Safari/537.36" ECDHE-RSA-AES128-GCM-SHA256 TLSv1.2 arn:aws:elasticloadbalancing:us-east-2:123456789012:targetgroup/my-targets/73e2d6bc24d8a067 "Root=1-58337327-72bd00b0343d75b906739c42" "-" "-" 1 2018-07-02T22:22:48.364000Z "redirect" "https://example.com:80/" "-"
