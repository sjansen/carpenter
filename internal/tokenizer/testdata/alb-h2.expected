{
  "actions_executed": "redirect",
  "chosen_cert_arn": "-",
  "client_ip": "10.0.1.252",
  "client_port": "48160",
  "domain_name": "-",
  "error_reason": "-",
  "lb": "app/my-loadbalancer/50dc6c495c0c9188",
  "lb_status_code": "200",
  "matched_rule_priority": "1",
  "received_bytes": "5",
  "redirect_url": "https://example.com:80/",
  "request_creation_time": "2018-07-02T22:22:48.364000Z",
  "request_processing_time": "0.001",
  "request_proto": "HTTP/2.0",
  "request_url": "https://10.0.2.105:773/",
  "request_verb": "GET",
  "response_processing_time": "0.003",
  "sent_bytes": "257",
  "target_group_arn": "arn:aws:elasticloadbalancing:us-east-2:123456789012:targetgroup/my-targets/73e2d6bc24d8a067",
  "target_ip": "10.0.0.66",
  "target_list": "",
  "target_port": "9000",
  "target_processing_time": "0.002",
  "target_status_code": "200",
  "target_status_code_list": "",
  "timestamp": "2018-07-02T22:23:00.186641Z",
  "tls_cipher": "ECDHE-RSA-AES128-GCM-SHA256",
  "tls_protocol": "TLSv1.2",
  "trace_id": "Root=1-58337327-72bd00b0343d75b906739c42",
  "type": "h2",
  "user_agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36  (KHTML, like Gecko) Chrome/51.0.2704.103 Safari/537.36"
}
